group_project/group_activity.py

```diff
@@ -477,4 +477,3 @@ def import_xml_file(cls, file_name):
     def import_xml_string(cls, xml, grading_override = False):
         doc_tree = ET.fromstring(xml)
         return cls(doc_tree, grading_override)
-
```

Comment:
@marjev is it intentional? Looks like it would result in empty line at the end of file, so pep8 should criticize you for that.